### DISCUSSION TITLE
Ideas for sharing experiments

### DISCUSSION THREAD

**Discussion by slinlee** [PathmindAI/pathmind-webapp#3423] [Ideas]
@ejunprung check out these ideas from @fionnachan . We'll have to see how ppl use the first version;

> Wow my imagination is running wild!
> 
> We can add a profile page for each user to let other users check out their publicly shared experiments.
> 
> And private experiments shared among team members.
> 
> Also a list of favourite public experiments on the profile page....
> 
> Our model templates could be on the "Team Pathmind" profile and are forkable so they can start from there 😎

_Originally posted by @fionnachan in https://github.com/SkymindIO/pathmind-webapp/issues/3390#issuecomment-888840653_

**Comment by ejunprung** [comment]
Yup, I'd wait and see how people use the first version before building it out.

But first impression, I think the publicly shared experiments will be really good for sales/marketing. Kind of like reward function inspiration for all Pathmind users. The private ones will probably convince organizations to purchase a paid plan for collaboration reasons so these all sound good to me.
